cmd/validator/flags/flags.go

```diff
@@ -340,7 +340,7 @@ var (
 	// SuggestedFeeRecipientFlag defines the address of the fee recipient.
 	SuggestedFeeRecipientFlag = &cli.StringFlag{
 		Name:  "suggested-fee-recipient",
-		Usage: "Sets ALL validators' mapping to a suggested an eth address to receive gas fees when proposing a block. Overrides the --fee-recipient-config-file flag if set",
+		Usage: "Sets ALL validators' mapping to a suggested an eth address to receive gas fees when proposing a block. Setting can be overridden",
```

Comment:
Perhaps it'd be good to clarify and amend can be overridden by "what"?

Reply:
will add, it was just awkward since this pr doesn't have the proposer setting flag changes